moninag/tests/unittests/registration/test_view.py

```diff
@@ -0,0 +1,316 @@
+import json
+
+from django.core.urlresolvers import reverse
+from django.test import TestCase, Client, mock
+
+from registration.models import CustomUser
+
+
+class Fakeuidb64(object):
+    """Mock uidb64 for password reset"""
+
+    def __init__(self, *args, **kwargs):
+        pass
+
+    def uidb64(self):
+        return '15'
```

Comment:
uidb64 get's the user's ID in the view. Therefore i mock it to return 15, which is an id of tested user :upside_down_face: